src/main/state/index.js

```diff
@@ -36,21 +26,61 @@ function onLoadState(key) {
   return decryptFromFile(key, storeFileEncryptionAlgorithm, storeFilePath);
 }
 
+/**
+ * Create a onSaveState handler.
+ * Used to keep track of pending save requests that must be fulfilled before exiting the app.
+ */
+function OnSaveState() {
```

Comment:
why is it capitalized ?

Reply:
oh it is a class.
you should add a @constructor (or @class) tag in the jsdoc

Reply:
I added `@class`, eslint didn't like the `@constructor`.